e2e_correctness/create_test/test_set_property/test.yml

```diff
@@ -0,0 +1,6 @@
+memgraph_query: >
+    MATCH (n)
+    WHERE n.id=3
+    CALL graph_updates.set_property(n, "new_prop", "2") YIELD * RETURN *;
+
+neo4j_query: MATCH (n) WHERE n.id=3    CALL apoc.create.setProperty(n, "new_prop", "2") YIELD node RETURN node;
```

Comment:
Just format this (and other Neo4j queries in the tests)

Reply:
I will delete these tests, I just want to see them pass

Reply:
No worries